web/src/frontend/src/digisos/start/Bosted.tsx

```diff
@@ -40,10 +41,14 @@ class Bosted extends React.Component<
 
 	componentDidMount() {
 		this.props.dispatch(resetSoknad());
```

Comment:
burde resetSoknad i tillegg resette faktaene? Nå brukes vel kun soknad-kallene til å hente faktaene så er kanskje naturlig om den resettes også derfra?

Reply:
Svarer her jeg -  opprettSoknad-action inneholder en del ting som kun er spesifikt for digisos mens hentsoknad og resetsoknad er uavhengig så de kan vi hvertfall flytte og burde nok flytte også. Så kan man se på om man får trekt ut det fra opprettSoknad ut slik atd en også kan flyttes